--- apertium-es-ro-0.7.1/Makefile.am.orig	2011-01-02 14:58:35.629905720 +0100
+++ apertium-es-ro-0.7.1/Makefile.am	2011-01-02 15:10:59.653898177 +0100
@@ -86,7 +86,7 @@
 	$(PREFIX1).autopgen.bin $(PREFIX2).autopgen.bin \
 	$(PREFIX1).prob $(PREFIX2).prob trules-$(PREFIX1).xml \
 	trules-$(PREFIX2).xml trules-$(PREFIX1).bin \
-	trules-$(PREFIX2).bin $(PREFIX1).mode $(PREFIX2).mode
+	trules-$(PREFIX2).bin $(PREFIX2).mode
 
 install-data-local:
 	apertium-gen-modes modes.xml apertium-$(PREFIX1)
