--- apertium-es-ro-0.7.3/configure.ac.orig	2015-12-29 16:58:02.972654803 +0100
+++ apertium-es-ro-0.7.3/configure.ac	2015-12-29 17:16:25.705941861 +0100
@@ -32,8 +32,13 @@
                   APERTIUM_VER=31
               ],
               [
-                  PKG_CHECK_MODULES(APERTIUM, apertium-3.2 >= 3.2.0)
+                  PKG_CHECK_MODULES(APERTIUM, apertium-3.2 >= 3.2.0,
+		  [
                   APERTIUM_VER=32
+		  ],
+		  [PKG_CHECK_MODULES(APERTUM, apertium >= 3.3.0)
+		   APERTIUM_VER=3
+		  ])
               ])
           ])
       ])
