--- apertium-es-ro-0.7.1/Makefile.am.orig	2007-10-11 21:06:30.000000000 +0200
+++ apertium-es-ro-0.7.1/Makefile.am	2011-01-02 12:27:14.809901529 +0100
@@ -91,8 +91,8 @@
 install-data-local:
 	apertium-gen-modes modes.xml apertium-$(PREFIX1)
 # spanish -> romanian is not currently functional ~fmt
-#	$(INSTALL_DATA) $(PREFIX1).mode $(apertium_es_modesdir)
-	$(INSTALL_DATA) $(PREFIX2).mode $(apertium_es_modesdir)
-	$(INSTALL_DATA) $(BASENAME).trules-$(PREFIX2).xml $(BASENAME).trules-$(PREFIX1).xml $(apertium_es_rodir)
+#	$(INSTALL_DATA) $(PREFIX1).mode $(DESTDIR)$(apertium_es_modesdir)
+	$(INSTALL_DATA) $(PREFIX2).mode $(DESTDIR)$(apertium_es_modesdir)
+	$(INSTALL_DATA) $(BASENAME).trules-$(PREFIX2).xml $(BASENAME).trules-$(PREFIX1).xml $(DESTDIR)$(apertium_es_rodir)
 
 CLEANFILES = -rf $(TARGETS) modes
